africanus/gridding/perleypolyhedron/gridder.py

```diff
@@ -0,0 +1,85 @@
+import numpy as np
+from numba import jit, literally, prange
```

Comment:
```suggestion
import numba
from africanus.numba.util import jit
```

Then use

```python
numba.literally(x) #
for i in numba.prange(y):
   ...
```

in the code.

readthedocs breaks on numba decorators so they're faked in a readthedocs environment. If you're missing a decorator you should add one to ``africanus.utils.numba``

Other things like ``numba.float32`` or ``numba.literally`` should be accessed as attributes of an ``import numba``